BaseDataContactsSQL.cs

```diff
@@ -0,0 +1,137 @@
+﻿using Microsoft.Data.Sqlite;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace test1
+{
+    /// <summary>
+    /// первым делом вызови InitializationBD
+    /// </summary>
+    public class BaseDataContactsSQL : IDataContactInterface
+    {
+        private string _dataSourceBD = "";
```

Comment:
Лучше использовать так:
```suggestion
        private string _dataSourceBD = string.Empty;
```

Reply:
Поправил